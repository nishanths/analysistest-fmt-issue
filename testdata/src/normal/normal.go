package normal

var x = 0 // want "^removing$"
var y = 1 // want "^removing$"
