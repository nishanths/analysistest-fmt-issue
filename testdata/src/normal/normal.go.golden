package normal

// want "^removing$"
// want "^removing$"
